dali/python/nvidia/dali/types.py

```diff
@@ -520,9 +526,8 @@ def _convert(x, type):
         call_args["name"] = name
         del constructor_args["name"]
 
-    op = ops.Constant(device=device, fdata=fdata, idata=idata,
-                      shape=shape, dtype=dtype, layout=layout,
-                      **constructor_args)
+    op = ops.Constant(device=device, fdata=fdata, idata=idata, shape=shape, dtype=dtype,
+                      layout=layout, **constructor_args)
```

Comment:
Revert? The line breaks were placed so that they reflect semantics. Alternatively, use 1 argument per line.
```suggestion
    op = ops.Constant(device=device, fdata=fdata, idata=idata,
                      shape=shape, dtype=dtype, layout=layout, **constructor_args)
```